res/qml/Deck.qml

```diff
@@ -326,6 +326,54 @@ Item {
             activeColor: Theme.deckActiveColor
         }
 
+        Row {
+            anchors.left: playButton.right
+            anchors.leftMargin: 10
+            anchors.bottom: playButton.bottom
+            anchors.topMargin: 5
+            spacing: -1
+
+            Skin.IntroOutroButton {
+                keyPrefix: "intro_start"
+                group: root.group
+
+                text: "Intro\nStart"
```

Comment:
What do you think about this?

```suggestion
                text: '◢'
```

Reply:
I started with this and it didn't render so well in the UI. Overall, the whole QML design probably needs a good rework anyway (see Zulip) so I thought I'll go with the more explicit now but happy to keep this char if we think it's better

Reply:
Hmmm not a big fan
![image](https://github.com/user-attachments/assets/0b05464b-a8ac-475d-a1db-225e72e06b77)

Wdyt? Shall I keep with text till we can to properly design this?

Reply:
I think the icon looks weird, but better than the text ;-) but I leave it up to you, we need to improve the UI later on anyway (Maybe have a frame around both buttons, put the intro icon as the group label, and then have something like `|-` and `-|` as button labels or something like that).

Reply:
Yeah, for now I've been only focusing on feature, since I'm also working on a whole design project for QML so didn't want to waste time with temporary measure for this

Reply:
Yup. I think the waveforms and the library are the two major issues that prevent me from using in in a set. Most other stuff is working from my controller anyway 😁

Reply:
Yup. I think the waveforms and the library are the two major issues that prevent me from making qml actually usable in a set. Most other stuff is working from my controller anyway 😁